src/litserve/specs/openai.py

```diff
@@ -394,9 +394,9 @@ async def streaming_completion(self, request: ChatCompletionRequest, pipe_respon
 
             # Only use the last item from encode_response
             usage_info = sum(usage_infos)
-            chunk = ChatCompletionChunk(model=model, choices=choices, usage=None).json()
+            chunk = ChatCompletionChunk(model=model, choices=choices, usage=None)
             logger.debug(chunk)
-            yield f"data: {chunk}\n\n"
+            yield f"data: {json.dumps(chunk.model_dump())}\n\n"
```

Comment:
`model_dump_json()` is the recommended way from Pydantic. 

```suggestion
            yield f"data: {chunk.model_dump_json()}\n\n"
```

Reply:
thx, perfect